Source/FulcrumFS.Videos/VideoSourceValidationOptions.cs

```diff
@@ -0,0 +1,168 @@
+namespace FulcrumFS.Videos;
+
+#pragma warning disable SA1513 // Closing brace should be followed by blank line
+
+/// <summary>
+/// Options for validating a source video during processing.
+/// </summary>
+public class VideoSourceValidationOptions

```

Comment:
As per our discussion: should be split this into `VideoStreamValidationOptions` and `AudioStreamValidationOptions` instead or do you think it's better to leave it like this? I'm leaning towards splitting it.